search/query_params.hpp

```diff
@@ -123,6 +126,7 @@ class QueryParams
   vector<Token> m_tokens;
   Token m_prefixToken;
   bool m_hasPrefix = false;
+  bool m_isCategorialRequest = false;
```

Comment:
I propose you to rename this to isCategorialRequest.

Reply:
What do you mean? Remove the "m_"? It's against the style guide.